localnet/poktrolld/config/supplier3_stake_config.yaml

```diff
@@ -6,6 +6,10 @@ services:
     endpoints:
```

Comment:
Reminder to outline an `/e2e` test + GH issue to make sure it's in the codebase :)

Reply:
https://github.com/pokt-network/poktroll/blob/cad18f10991c98876b69b7953afd9e9fef777bd3/pkg/relayer/proxy/async.go#L62